frontend/src/components/RideModal/Pages/RiderInfo.tsx

```diff
@@ -38,16 +38,8 @@ const RiderInfoPage = ({ formData, onBack, onSubmit }: ModalPageProps) => {
     }, {});
     setNameToId(nameToIdObj);
 
-    fetch('/api/locations?active=true', withDefaults())
-      .then((res) => res.json())
-      .then(({ data }: { data: Location[] }) => {
-        const locationToIdObj = data.reduce((acc: ObjectType, l) => {
-          acc[l.name] = l.id;
-          return acc;
-        }, {});
-        setLocationToId(locationToIdObj);
-      });
-  }, [withDefaults, riders]);
+    setLocationToId(locations.map((l) => l.id));
```

Comment:
`locationToId` is a map from location name to location id, keep the original code from lines 44 to 48